### PR TITLE
added bourbon mule recipe and image

### PR DESCRIPTION
**Pull request by brittanyjoiner15**
Hope this works!

### CODE REVIEW

File: src/recipes/bourbon-mule.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "1/2 cup light ginger beer, such as Fever-Tree Naturally Light",
+        "1 fresh mint sprig",
+        "1 lime slice"
+    ],
+    "directions": [
+        "In a copper mug or highball glass, combine bourbon and lime juice.",
+        "Fill cup with ice. Add ginger beer; stir gently to combine.",
+        "Garnish with mint sprig and lime slice."
+    ],
+    "image": "<bourbon-mule.jpg>",
```
Comment (alfg) [inline src/recipes/bourbon-mule.json]:
```suggestion
    "image": "bourbon-mule.jpg",
```

File: src/recipes/bourbon-mule.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "source": "https://www.cookinglight.com/recipes/bourbon-mule",
+    "keywords": [
+        "bourbon mule",
+        "bourbon",
```
Comment (alfg) [inline src/recipes/bourbon-mule.json]:
```suggestion
        "bourbon"
```

**Comment by brittanyjoiner15** [comment]
Okay i made those edits and commited again (see note)
image


Do i need to do another pull request? (I'm new to this!)

**Comment by alfg** [comment]
> 
> 
> Okay i made those edits and commited again (see note)
> image
> 
> Do i need to do another pull request? (I'm new to this!)

No, you can keep adding commits to this PR and it will auto-update! :)

**Comment by alfg** [comment]
Looks good! Thanks for contributing!

**Pull request merged**